x/mongo/driver/topology/connection.go

```diff
@@ -292,10 +294,16 @@ func (c *connection) readWireMessage(ctx context.Context, dst []byte) ([]byte, e
 	if err != nil {
 		// We closeConnection the connection because we don't know if there are other bytes left to read.
 		c.close()
+		var message string
+		if err == io.EOF {
+			message = "Socket was unexpectedly closed"
```

Comment:
Error messages should generally use lowercase letters.

Reply:
done